packages/terra-status-view/src/StatusView.jsx

```diff
@@ -99,7 +100,7 @@ const StatusView = ({
   } else if (variant && !isGlyphHidden) {
     glyphSection = (
       <div className={cx('glyph')}>
-        <svg className={cx(variant)} />
+        <div className={cx(variant, 'bg-common')} />
```

Comment:
We should use a more descriptive class name here, glyph-icon, glyph-container, icon or something more descriptive. bg-common feels a little too generic, almost like a boostrap/global css value

Reply:
Changed b469734